Cargo.toml

```diff
@@ -4,3 +4,7 @@ members = [
     "druid-shell",
     "druid-derive",
 ]
+
+[patch.crates-io]
+piet-common = { git = "https://github.com/hwchen/piet", branch = "raw_d2d" }
```

Comment:
I know I already mentioned it, but just to make sure it's in the record, this will change to piet 0.0.9 as soon as that release happens. We'll coordinate the releases so they happen in succession.